tests/unit/test_celery.py

```
@@ -405,20 +430,40 @@ def test_bump_worker_loss_count_disabled_without_kv(self):
 			with patch.object(app.backend, 'get', side_effect=NotImplementedError, create=True):
 				self.assertEqual(bump_worker_loss_count('task-abc'), 0)
 
-	def test_abandon_task_returns_failure_error(self):
-		"""Abandoning returns results with a self-owned FAILURE Error so the chord proceeds."""
+	def test_abandon_task_persists_failure_error_to_store(self):
+		"""Abandoning returns topology-only, persisting a self-owned FAILURE Error to the store
+		(not the payload) so the chord proceeds and the failure is still queryable."""
+		import tempfile
+		from pathlib import Path
 		from secator.tasks import httpx
```

Flake8 (7.3.0)

[error] 'secator.tasks.httpx' imported but unused

(F401)